src/clib/CMakeLists.txt

```diff
@@ -9,139 +9,54 @@ find_package(Python COMPONENTS Interpreter Development.Module NumPy REQUIRED)
 find_package(SWIG 3.0.1 COMPONENTS REQUIRED)
 include(UseSWIG)
 
-# CMP0053: Simplify variable reference and escape sequence evaluation.
-# The OLD behavior for this policy is to honor the legacy behavior for
-# variable references and escape sequences.
-if (POLICY CMP0053)
-  cmake_policy(SET CMP0053 OLD)
-endif()
-
-# ======================================================================================
-# Looking for numpy; need to avoid that weird old system numpy is used (aka RHEL6)
-# THIS WAS A PAIN TO DEBUG
-#
-# Kudos https://github.com/Eyescale/CMake/blob/master/FindNumPy.cmake
-# via https://github.com/DeepLearnPhysics/larcv3/blob/src/CMakeLists.txt
-# ======================================================================================
-
-if(NOT Python_EXECUTABLE)
-  if(NumPy_FIND_QUIETLY)
-    find_package(PythonInterp QUIET)
-  else()
-    find_package(PythonInterp)
-    set(__numpy_out 1)
-  endif()
-endif()
-
-if (Python_EXECUTABLE)
-  # Find out the include path
-  execute_process(
-    COMMAND @PYTHON_EXECUTABLE@ -c
-    "try: import numpy; print(numpy.get_include(), end='')\nexcept:pass\n"
-    OUTPUT_VARIABLE __numpy_path)
-  # And the version
-  execute_process(
-    COMMAND @PYTHON_EXECUTABLE@ -c
-            "try: import numpy; print(numpy.__version__, end='')\nexcept:pass\n"
-            OUTPUT_VARIABLE __numpy_version)
-    message(STATUS "Detected NumPy: ${__numpy_path}; version ${__numpy_version}")
-elseif(__numpy_out)
-  message(ERROR "Python executable not found.")
-endif(Python_EXECUTABLE)
-
-find_path(PYTHON_NUMPY_INCLUDE_DIR numpy/arrayobject.h
-  HINTS "${__numpy_path}" "${PYTHON_INCLUDE_PATH}" NO_DEFAULT_PATH)
-
-if(PYTHON_NUMPY_INCLUDE_DIR)
-  set(PYTHON_NUMPY_FOUND 1 CACHE INTERNAL "Python numpy found")
-endif(PYTHON_NUMPY_INCLUDE_DIR)
-
-message(STATUS "XTGeo Python include path: ${PYTHON_INCLUDE_PATH}")
-message(STATUS "XTGeo Python include dirs: ${PYTHON_INCLUDE_DIRS}")
-message(STATUS "XTGeo Python executable  : ${PYTHON_EXECUTABLE}")
-message(STATUS "XTGeo numpy include path : ${PYTHON_NUMPY_INCLUDE_DIR}")
-
-# ======================================================================================
-# Find swig
-# ======================================================================================
-
 # in case required SWIG is missing or too old and we are within Equinor, need a path to
 # a SWIG version that works (RHEL7 have SWIG 2 by default which is too old)
 if("$ENV{HOSTNAME}" MATCHES "statoil" AND CMAKE_SYSTEM_NAME MATCHES "Linux")
   LIST(APPEND CMAKE_PROGRAM_PATH "/prog/res/opt/rhel7/swig_4.0.2/bin")
   message(STATUS "Using Swig from /prog/res")
 endif()
```

Comment:
Since we now have included `swig` via pip in `pyproject.toml`, I think we can delete this. (At the time when this was made, swig did not exists as a pip package)

Reply:
Great! Done 😄